.github/workflows/build.yml

```diff
@@ -42,6 +42,13 @@ jobs:
           key: ${{ runner.os }}-konan-${{ github.sha }}
           restore-keys: ${{ runner.os }}-konan-
 
+      - name: Archive ktlint results
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: ktlint-results
+          path: "**/build/reports/ktlint/"
```

Comment:
So, you will want to modify this to actually archive test results/output files that would be beneficial for diagnosing failures after the fact

Reply:
Ktlint is a printer for Kotlin code, that utility isn't relevant for the iOS repo